### PR TITLE
Improve refresh script to reduce runtime and eliminate occasional mis…

### PR DESCRIPTION
**Pull request by kareemhamdy100**
…counting of Issue, PRs #20 

the main problem is that you  access database during fetch data from api 

access database should done  one time after you get all users data  so I made that  and try it for 
one repo  "Rocket.chat" and the run time down from 7 minut to  3 minut in my machine  

I hope I found away to reduce the updating database in less time 
my suggestion to update the showing data without store it  and  every one hour update the database that's maybe a good solution

### CODE REVIEW

**Comment by shubhsherl** [comment]
@kareemhamdy100 Can you run it at least for 4 or 5 repos. And while running check if all the API calls are `200 Success` ? I guess the main problem is that some calls are getting response of `403 Rate limit`.

**Comment by Sing-Li** [comment]
@kareemhamdy100  comment here ... let us know what improvement your latest update includes!

**Comment by kareemhamdy100** [comment]
my code update all list in only 6 min

about 40 request per min so it never access 5000 request in hour

**Comment by kareemhamdy100** [comment]
Improve :in first time it will get the all data in 8 ~11 min and after that will make refresh depend on the number of "open_issues" filed in each repo
 which refer to 
open issues +open PR if any of them changed the next request will update things related to the changed repo only

for the largest repo in rocketChat is Rocket.chat  it's take about 2 min all others repos it take less than 1 min

File: core/views.py [hunk 1]
```diff
@@ -44,54 +45,56 @@ def getOrganizationRepositories(org, url=''):
...
+            if currRepo[0].openIssues != repo_['open_issues'] and currRepo[0].include:
+                currRepo.update(openIssues=repo_['open_issues'])
+                issueRepos += currRepo
 
-    repo = Repository.objects.filter(include=True)
-    includedRepo = json.loads(serializers.serialize('json', list(repo), fields=('owner','repo')))
-
-
-    for repo_ in includedRepo:
+    forIssues = json.loads(serializers.serialize('json', list(issueRepos), fields=('owner', 'repo')))
```
Comment (shubhsherl) [inline core/views.py:60]: Should also consider the count of PRs.

**Pull request closed without merging**